pkg/apis/operator/v1alpha1/tektonconfig_defaults.go

```diff
@@ -30,6 +30,8 @@ func (tc *TektonConfig) SetDefaults(ctx context.Context) {
 
 	setAddonDefaults(&tc.Spec.Addon.Params)
 
+	setHubDefaults(&tc.Spec.Hub.Params)
```

Comment:
defaulting through webhook would mean this param appears on k8s platform too
we did for addon through webhook as there is a plan to enable it on k8s
but having this on k8s too might confuse the upstream users

Reply:
Yes @sm43 This is what discussed in the call and decided to not do defaulting, only validation is enough.

Reply:
Fixed